.circleci/config.yml

```diff
@@ -734,6 +774,22 @@ workflows:
       - binary_windows_conda:
           name: binary_windows_conda_py3.8
           python_version: '3.8'
+      - build_docs:
+          name: build_docs
+          python_version: '3.8'
+          requires:
+          - binary_linux_wheel_py3.8
+      - upload_docs:
+          filters:
+            branches:
+              only:
+              - nightly
+            tags:
+              only: /v[0-9]+(\.[0-9]+)*-rc[0-9]+/
```

Comment:
Sorry for making comment after the merge, but this means that release documentation is only pushed on release tag, right?
How about changing this to `release/*` branch so that even after the release, we can fix the documentation by pushing doc change to `release/*` branch?
@seemethere @brianjo @mattip

Reply:
The way I see it is like this:
- gh-pages/master is built every night and documents the latest and greatest HEAD of the development
- gh-pages/0.8 and all the numbered directories document the latest release; what end users would download to their machines if they install from conda or pip and specified a version.
- stable points to the latest numbered directory, what users would get if they conda or pip install and do not specify a version.

Bugs in the release would indeed be committed to the release/* branch via PRs, these would trigger the `build_docs` build and you could examine the artifacts\*. But the new documentation would only become "official" when the release is tagged and the fix is available to the general public as a major/minor/patch release, so a tag event is when the contents of the numbered directory should change.

You could fix bugs in the documentation that are discovered after release by building documentation locally and pushing the built documentation to the correct directory (using the steps in `.circleci/build_docs/commit_docs.sh` or calling the script to do it for you).

\* I should improve the CI process and make the artifacts available for viewing after the `build_docs` step.